src/lambda/routes/invocations/InvocationsController.js

```diff
@@ -74,6 +74,14 @@ export default class InvocationsController {
         // RequestTooLargeException, InvalidParameterValueException,
         // and whatever response is thrown when the response is too large.
       }
+
+      // Checking if the result of the Lambda Invoke is a primitive string to wrap it. this is for future post-processing such as Step Functions Tasks
```

Comment:
It would be great to cover it with a test case

Reply:
I added a couple of unit tests to cover this case and when the response of an Invoked Lambda is a JSON. let me know if we require more coverage 👌

Reply:
@pgrzesik I added a couple of unit tests to cover this case and when the response of an Invoked Lambda is a JSON. let me know if we require more coverage 👌